src/js/apps/patients/patient/patient_app.js

```diff
@@ -42,6 +42,11 @@ export default SubRouterApp.extend({
     return Radio.request('entities', 'fetch:patients:model', patientId);
   },
 
+  onFail() {
+    Radio.trigger('event-router', 'notFound');
+    this.stop();
+  },
+
```

Comment:
After a patient is archived in a specific workspace, retrieving that patient returns a `410` error from the api.

In that scenario, this shows the `404` page error. Not sure what we want to show.

<img width="347" alt="Screenshot 2024-01-26 at 2 36 34 PM" src="https://github.com/RoundingWell/care-ops-frontend/assets/35355575/1c7ba375-def7-4999-8503-faf3259b7e1f">

Reply:
Yep. so with our api 404 means that api endpoint doesn't exist.  410 means it's not found on an existing api endpoint.  So this is what we want to show

Reply:
could do: `onFail(options, { response }) { if (response.status === 410`

Reply:
Updated this to do the `if (response.status === 410)`.